src/OverflowVisibleContainer/OverflowVisibleContainer.tsx

```diff
@@ -41,7 +41,8 @@ export const OverflowVisibleContainer = React.forwardRef<Ref, Props>(
 				<div
 					ref={ref}
 					className={cn("kit-overflow-visiblecontainer", className, {
-						"kit-overflow-visiblecontainer_fixed": isFixed
+						"kit-overflow-visiblecontainer_fixed": isFixed,
+						"kit-overflow-visiblecontainer_hidden": !hidden
```

Comment:
Вадим, ну тут-то уже не нужно отрицание, раз мы модификатор поменяли на `_hidden`)
```suggestion
						"kit-overflow-visiblecontainer_hidden": hidden
```

Reply:
запутался что-то